lms/static/sass/views/_account-settings.scss

```diff
@@ -551,9 +551,10 @@
         margin: 20px 0;
       }
 
-      .paragon__modal-open {
+      .paragon__modal {
         overflow-y: scroll;
         color: $dark-gray;
+        margin-top: 0.5rem;
 
         .paragon__modal-title {
           font-weight: $font-semibold;
```

Comment:
On line 583 below, the btn primary style is imported for any button that's not the cancel button. I think you could instead just pass "primary" as a `buttonType` to the buttons in the Modal JSX. [See the modal story that does this](https://github.com/edx/paragon/blob/master/src/Modal/Modal.stories.jsx#L80).